homework_9/9.2_task.py

```diff
@@ -0,0 +1,85 @@
+from datetime import date
+
+class Employee:
+    def __init__(self, name_surname: list, age: int, position: str, salary: int):
+        self.__name_surname = name_surname
+        self.__age = age
+        self.__position = position
+        self.__salary = salary
+
+    @classmethod
+    def date_of_birth(cls, name_surname: list, date_of_birth: date, position: str, salary: int):
+        """If you have date of birth instead of age this function calculates the age"""
+        today = date.today()
+        age = today.year - date_of_birth.year - ((today.month, today.day) < (date_of_birth.month, date_of_birth.day))
+        return f'{" ".join(name_surname)} is {age}. {" ".join(name_surname)} is {position} with salary {salary}$'
+
+    @property
+    def name_surname(self):
+        return self.__name_surname
+
+    @name_surname.setter
+    def name_surname(self, new_name_surname: list):
+        if isinstance(new_name_surname, list):
+            self.__name_surname = new_name_surname
+
+    @name_surname.deleter
+    def name_surname(self):
+        del self.__name_surname
+
+    @property
+    def age(self):
+        return self.__age
+
+    @age.setter
+    def age(self, new_age: int):
+        if new_age > 18:
+            self.__age = new_age
+        else:
+            raise TypeError("We can't hire non adult")
+
+    @age.deleter
+    def age(self):
+        del self.__age
+
+    @property
+    def position(self):
+        return self.__position
+
+    @property
+    def salary(self):
+        return self.__salary
+
+    @position.setter
+    def position(self, new_position: str):
+        if isinstance(new_position, str):
```

Comment:
lets implement some CONSTANT in this class, and positions can be set from it only.